resources/views/auth/login/cover.blade.php

```diff
@@ -10,7 +10,7 @@
                 @include(backpack_view('auth.login.inc.form'))
                 @if (config('backpack.base.registration_open'))
                     <div class="text-center text-muted mt-3">
-                        <a href="{{ route('backpack.auth.register') }}" tabindex="-1">{{ trans('backpack::base.register') }}</a>
+                        <a tabindex="6" href="{{ route('backpack.auth.register') }}">{{ trans('backpack::base.register') }}</a>
                     </div>
                 @endif
```

Comment:
This bit of code is the same on all 3 layouts (cover, default and illustration). 

Wouldn't be better to make it a partial and include it ? `@includeWhen(config('backpack.base.registration_open'))`

Reply:
Neeah... too small to split out